styles/app.scss

```diff
@@ -4,3 +4,291 @@
 $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome';
 @import 'box-sizing';
+@import 'container';
+
+$brand-primary: #3473b5;
+$brand-white: #fff;
+$pink: #e577aa;
+$blue: #72bbe9;
+$green: #7abe93;
+$yellow: #e5b178;
+
+$gutter: 1rem;
+$max-contentwidth: 60rem;
+
+
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.grid-item {
+  display: flex;
+  width: calc(100% / 3);
+  padding: $gutter;
+}
+
+.shop-item {
+  border: 0.3rem solid $brand-primary;
+}
+
+.shop-item-buttons {
+  display: flex;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  border-top: 0.3rem solid $brand_primary;
+
+  :nth-child(5n) {
+    border-right: 0;
+  }
+}
+
+.shop-item-description {
+  padding: $gutter;
+}
+
+.shop-button {
+  width: calc(100% / 5);
+  padding: 0;
+
+  border-right: 0.2rem solid $brand_primary;
+  color: $brand_primary;
+  text-align: center;
+
+  &:hover {
+    color: $brand_white;
+    background: $brand-primary;
+  }
+}
+
+
+.shirt-div {
+  width: 100%;
+  padding-top: $gutter;
+  padding-right: $gutter;
+  padding-left: $gutter;
+
+}
+
+img {
+  width: 100%;
+}
+
+li {
+  list-style: none;
+}
+
+.shop-item-title {
+  text-align: center;
+  color: $brand-primary;
+  padding-top: $gutter;
+  padding-right: $gutter;
+  padding-left: $gutter;
+
+}
+
+.shop-item-description {
+  color: $brand-primary;
+  text-align: center;
+}
+
+.shop-button {
+  list-style: none;
+}
+
+.dropdown {
+  position: relative;
+}
+
+.fa {
+  font-size: 1.9rem;
+  padding: 0.3rem;
+
+  &:hover {
+    font-size: 2.1rem;
+  }
+}
+
+.shirt-size {
+  font-size: 1.9rem;
+  padding: 0.3rem;
+}
+
+.dropdown__options {
+  display: none;
+  position: absolute;
+  text-align: center;
+  padding: 0;
+  width: 100%;
+
+  left: 0rem;
+  top: -11.5rem;
```

Comment:
Here I think the same idea applies as in your Tie site. Push from the closest edge. Also, look at using `%` units with pushing from edges to go past the edge of the parent.